diff --git a/dist/cjs/index.js b/dist/cjs/index.js
index 454012c491d38002a18d07d7d531b94a4a8a991e..4c59d048267d0cc53dc53e03e6f12d8585933647 100644
--- a/dist/cjs/index.js
+++ b/dist/cjs/index.js	
@@ -140,7 +140,7 @@ class DriverProcess {
                     stdio: ['ignore', 'pipe', 'pipe'],
                     detached: false,
                     env: spawnEnv,
-                    shell: process.platform === 'win32',
+                    shell: false,
                 });
                 log$6.info(`[${identifier}] Spawned process with PID: ${this._proc.pid ?? 'unknown'}`);
                 this.startupTimeout = setTimeout(() => {
diff --git a/dist/esm/index.js b/dist/esm/index.js
index 391d14a16f4c31525c6aa6d479b1702401dac1ad..7472ca2ac0ec89131ec4915f80721bc87cd42f4b 100644
--- a/dist/esm/index.js
+++ b/dist/esm/index.js	
@@ -138,7 +138,7 @@ class DriverProcess {
                     stdio: ['ignore', 'pipe', 'pipe'],
                     detached: false,
                     env: spawnEnv,
-                    shell: process.platform === 'win32',
+                    shell: false,
                 });
                 log$6.info(`[${identifier}] Spawned process with PID: ${this._proc.pid ?? 'unknown'}`);
                 this.startupTimeout = setTimeout(() => {
